tests/test_tags.py

```diff
@@ -0,0 +1,23 @@
+def test_default_tag(hlwm):
+    assert hlwm.get_attr('tags.count') == '1'
+    assert hlwm.get_attr('tags.0.name') == 'default'
+
+
+def test_add_tag(hlwm):
+    hlwm.callstr('add foobar')
+
+    assert hlwm.get_attr('tags.count') == '2'
+    assert hlwm.get_attr('tags.1.name') == 'foobar'
+
+
+def test_move_focused_client_to_new_tag(hlwm):
+    hlwm.callstr('add foobar')
+    (proc, winid) = hlwm.create_client()
+
+    hlwm.callstr('move foobar')
+
+    # TODO: Assert that foobar now has 1 client
```

Comment:
@t-wissmann How would I do these assertions? The tag's `client_count` attribute and the client's `tag` attribute don't seem to exist yet/anymore. Am I missing something?

Reply:
These should come back (tags.*.clients.count and clients.*.tag) as symlinks, right?

Reply:
So there is no way to do these assertions right now? Would you be fine with merging this like it is then? Better than nothing, I suppose.

Reply:
Yes, they don't exist yet, and I'm fine with merging this for the time being.